### ISSUE TITLE
CSP violations - bilingual error pages

### ISSUE THREAD

**Issue by patheard** [cds-snc/security-tools#47]
# Summary
Currently, the CSP violation HTTP errors pages are English only.  Ideally they should be shown in the user's selected language, but failing that, we can fall back to bilingual error pages:
```
images/csp_violation_report_service/app/templates/errors/403.html
images/csp_violation_report_service/app/templates/errors/404.html
images/csp_violation_report_service/app/templates/errors/500.html
images/csp_violation_report_service/app/templates/maintenance.html
```

**Comment by patheard** [comment]
CSP report service has been sunset.
